File: sprint2/By languages/python/lesson5code1.py
class Node:
    def __init__(self, value=None, next=None):
        self.value = value
        self.next = next
        
def print_linked_list(vertex):
		while vertex:
				print(vertex.value, end=" -> ")
				vertex = vertex.next
		print("None")
		
>>> n3 = Node('third')
>>> n2 = Node('second', n3)
>>> n1 = Node('first', n2)
>>> print_linked_list(n1)
first -> second -> third -> None
>>> print_linked_list(n2)
second -> third -> None

File: sprint2/By languages/swift/lesson5code1.swift
class Node<T> {
    var value: T?
    var next: Node<T>?
    
    init(value: T?, next: Node<T>? = nil) {
        self.value = value
        self.next = next
    }
}

func printLinkedList<T>(_ vertex: Node<T>?) {
    var currentVertex = vertex
    while let value = currentVertex?.value {
        print(value, terminator: " -> ")
        currentVertex = currentVertex?.next
    }
    print("None")
}

let n3 = Node(value: "third")
let n2 = Node(value: "second", next: n3)
let n1 = Node(value: "first", next: n2)
printLinkedList(n1)
// first -> second -> third -> None
printLinkedList(n2)
// second -> third -> None